Documentation/guides/k8s-aws-ansible/install.rst

```diff
@@ -0,0 +1,1430 @@
+*Kubernetes + CoreOS + Cilium Stack building with Ansible*
+
```

Comment:
Will be great to have a quick introduction section which describes the scope of the guide and what it achives

Reply:
Please also state assumptions or minimum requirements to proceed (e.g. aws cli installed, AWS credentials set up, etc)